presto-docs/src/main/sphinx/release/release-0.200.rst

```diff
@@ -0,0 +1,31 @@
+=============
+Release 0.200
+=============
+
+General Changes
+---------------
+
+* Disable early termination of inner/right joins when the right side has zero rows.
```

Comment:
So to confirm, left joins are not affected?

Reply:
Correct